CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.11 (February 14, 2024)
+ENHANCEMENTS
+* `Adds support for Datasets
+
```

Comment:
I think this should be a minor release, not a patch

Reply:
Updated